hasktorch/src/Torch/Typed/Functional.hs

```diff
@@ -1893,10 +1902,10 @@ allclose rtol atol equalNaN input other =
 -- (Int64,([4,1],[[1],[1],[1],[0]]))
 --
 -- >>> dtype &&& shape &&& (\t' -> D.asValue (toDynamic t') :: [Int]) $ argmax @0 @DropDim t
--- (Int64,([2],[3,1]))
+-- (Int64,([2],[0,1]))
```

Comment:
t is `fromJust [[0, 1], [-1, 2], [0, 1], [0, -2]] `
So the meaning of  `(Int64,([2],[3,1]))` is the same as `(Int64,([2],[0,1]))`'s one.

Reply:
thanks, I wonder why they changed the implementation of argmax :)

Reply:
https://github.com/pytorch/pytorch/pull/37864
Previous libtorch may return last index.